spec/models/solr_document_spec.rb

```diff
@@ -26,6 +26,14 @@
     end
   end
 
+  describe "rights_statement" do
+    let(:attributes) { { 'rights_statement_sim' => ['A rights statement'] } }
```

Comment:
I think the field you want is `rights_statement_tesim`, and so maybe it would be a good idea to check the return value.